### ISSUE TITLE
Replace text placeholder with somthing meaningful

### ISSUE THREAD

**Issue by pjanck** [bSI-InfraRoom/TI-Bridge#5]
https://github.com/bSI-InfraRoom/TI-Bridge/blob/f4ed353f05153684949b4a121c2b6ff1b6b1f188/IFC4x3/ModelViews/TI-Bridge/DocModelView.xml#L83

`dunno yet` should be populated with a better value.

**Comment by SergejMuhic** [comment]
No shit sherlock. 😄  This is obvious as hell.

The only dataset with alignmnet that I had was ACCA and I had no permission to use that. Should pay more attention at them meetings we keep having. 😉

**Comment by pjanck** [comment]
> This is obvious as hell.

I agree. I've opened an issue to not forget it. I assume the file will become bigger with more instructions being drafted.

**Comment by SergejMuhic** [comment]
Hm, my experience with issues is the opposite. They fill up the backlog and do not get closed by the people who should close them ... Let me open an issue on this.

EDIT: that is why I made it obvious.
EDIT2: look at https://github.com/bSI-InfraRoom/IFC-Specification/issues and https://github.com/buildingSMART/IFC4.3.x-development/issues - yes we are bad at it 😄
